core/src/execution/datafusion/expressions/scalar_funcs/chr.rs

```diff
@@ -44,10 +44,15 @@ pub fn chr(args: &[ArrayRef]) -> Result<ArrayRef> {
         .iter()
         .map(|integer: Option<i64>| {
             integer
-                .map(|integer| match core::char::from_u32(integer as u32) {
-                    Some(integer) => Ok(integer.to_string()),
-                    None => {
-                        exec_err!("requested character too large for encoding.")
+                .map(|integer| {
+                    if integer < 0 {
```

Comment:
@andygrove can we use `unlikely` (requires nightly) to indicate a less likely branch

Reply:
I thought we are moving to stable?